doc/api/process.md

```diff
@@ -1977,7 +1977,7 @@ A boolean value that is `true` if the current Node.js build includes support for
 ## `process.features.uv`
 
 <!-- YAML
-added: REPLACEME
+added: v22.9.0
```

Comment:
```suggestion
added: v0.5.3
```

https://github.com/nodejs/node/pull/54897/files#r1756590407

Reply:
I thought added referred to when the feature was actually added, not documented

Reply:
Can you either remove #54897 or include https://github.com/nodejs/node/pull/54982? Let’s not release something wrong – it’s doc-only, it shouldn’t require a full CI re-run.

Reply:
Removed.